Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```diff
@@ -1666,9 +1666,20 @@ script:
     - description: Comma-separated list of endpoint IDs. Can be retrieved by running the core-get-endpoints command.
       name: endpoint_ids
       required: true
-    - description: Comma-separated list of shell commands to execute.
+    - description: Comma-separated list of shell commands to execute. Use is_raw_command=false to prevent splitting by commas.
```

Comment:
```suggestion
    - description: Comma-separated list of shell commands to execute. Set the `is_raw_command` parameter to `true` to prevent splitting by commas. (Useful when using `||`, `&&` separators for controlling the flow of multiple commands).
```

Reply:
Fixed in the YML and REAME in commit df4e4271a26914a697650c106fa327f222428475